source/image-handler/package.json

```diff
@@ -8,24 +8,21 @@
   "version": "5.1.0",
   "private": true,
   "dependencies": {
-    "color": "3.1.2",
+    "color": "3.1.3",
     "color-name": "1.1.4",
-    "sharp": "^0.26.3"
+    "sharp": "^0.28.1"
   },
   "devDependencies": {
-    "aws-sdk": "2.712.0",
-    "aws-sdk-mock": "^5.1.0",
+    "aws-sdk": "2.887.0",
     "jest": "^26.4.2",
-    "mocha": "^8.1.3",
-    "nyc": "^15.1.0",
-    "sinon": "^9.0.3"
+    "sinon": "^10.0.0"
```

Comment:
Do the tests still work? Sometimes bumping sinon or jest related stuff breaks it.

Reply:
this stuff is from `upstream`

Reply:
but yeah, the tests were still green. I had to change only some binary fixtures since the actual image handler library wa also updated (sharp)